inspec.yml

```diff
@@ -192,97 +192,52 @@ inputs:
     value: "I've read & consent to terms in IS user agreem't."
 
   # SV-204404
-  - name: expected_lock_delay
-    desc: Expected value for the screensaver lock-delay
+  - name: lock_delay
+    desc: Maximum value for the screensaver lock-delay
     type: Numeric
     value: 5
 
   # SV-204411
-  - name: expected_difok
-    desc: Number of characters that must be different from previous password
-    type: Numeric
-    value: 8
-
-  # SV-204411
-  - name: max_difok
+  - name: difok
     desc: Minimum number of characters that must be different from previous password, by policy
```

Comment:
This desc confuses me for some reason, the original was for `max_difok` and the check in the SV-204411 control is for lte, should this not read Maximum number ... ?

Reply:
According to the check text, the inspec.yml desc is correct, but the control was updated incorrectly in the PR. Changing the control to read:
`    its('difok') { should cmp >= input('difok') }`